server/lib/Room.js

```diff
@@ -2090,8 +2113,9 @@ class Room extends EventEmitter
 
 	async _getRouterId()
 	{
+		// @TODO: How do I fetch a router here?
```

Comment:
Please check this

Reply:
Same as before, probably you will need to remove this function.